src/cogent3/parse/sequence.py

```diff
@@ -90,6 +91,52 @@ def format_from_filename(filename, format=None):
         return filename[filename.rfind(".") + 1 :]
 
 
+def load_from_json(filename, classes):
+    """Loads objects from json files.
+
+    Parameters
+    ----------
+    filename: name of the json file
+    classes: A series of the Cogent3 types, for example: (Alignment, ArrayAlignment)
+
+    """
+    assert all(
+        (isinstance(klass, type) for klass in classes)
+    ), "classes should be a series of Cogent3 types, for example: (Alignment, ArrayAlignment)"
+
+    f = open_(filename)
+    content = json.loads(f.read())
+    f.close()
+    # loads from a record json file (generated by make_record_for_json)
+    if set(content.keys()) == {"identifier", "data", "completed"}:
```

Comment:
we can simplify the logic here. First, just use the `load_record_from_json` function since if the nature of the records changes, that's where that logic will be specified.

```python
try:
    _, data, completed = load_record_from_json(content)
    if not completed:
        raise TypeError("json file is a record for type NotCompleted.")
except KeyError:
    data = content
```
This logic produces a `data ` variable that should be the same if it comes from a db record or from directly writing results of `to_json()`.